samples/subsys/usb_c/sink/boards/b_g474e_dpow1.overlay

```diff
@@ -27,8 +27,28 @@
 			tcpc = <&ucpd1>;
 			vbus = <&vbus1>;
 			power-role = "sink";
+            		num-snk-pdos= <1>;
 			sink-pdos = <PDO_FIXED(5000, 100, 0)>;
+            		fr-swap-reqd-type-c-current-as-initial-source= <0>;
+			vconn-swap-to-off-supported;
+            		higher-capability-set;
 		};
 		/* usbc.rst usbc-port end */
 	};
+
+    vif {
+        #address-cells = <1>;
+        #size-cells = <0>;
+        usbcvif1: usbcvif@1 {
+            compatible = "vif";
+            reg = <1>;
+            vendor-name = "Zephyr";
+            model-part-number = "b_g474e_dpow1";
```

Comment:
there's standard `model` property. Why is this needed?

Reply:
`model-part-number` seemed to be different from model and more specific to USB-C component. Following is it's definition : "To be provided by the Vendor. This field should match what appears in the USB-IF Product Registration form. If the Qualifying Product has a Model Name entry in USB4 DROM, then this field should be identical to that entry. The USB-IF uses this field and /VIF/Product_Revision to generate the product name in the USB Integrator’s List"